### ISSUE TITLE
Traffic

### ISSUE THREAD

**Issue by schr3j** [chenasraf/gInbox#23]
87MB sent and 2,15GB (!!) received from gstatic.com
Is that a problem with inbox itself or with the wrapper?

![screenshot](https://cloud.githubusercontent.com/assets/1710904/5855748/e1113230-a23b-11e4-8893-8d60bf43e14d.png)

**Comment by cameronbracken** [comment]
Second that, just installed yesterday! 

![image](https://cloud.githubusercontent.com/assets/32280/5861558/a3b61392-a227-11e4-80f8-e8170e7cce8f.png)

**Comment by schr3j** [comment]
When Inbox is opened in Chrome there is not that much traffic. It seems to have something to do with the wrapper.

**Comment by chenasraf** [comment]
I'm not sure why it downloads that much data, if it did that because of the user agent, it shouldn't have done anything because I spoof Chrome's user agent (as you can see [here](https://github.com/chenasraf/gInbox/blob/0.2.3c/gInbox/WebViewController.swift#L20))... Maybe it's doing some kind of feature-detection? Did you guys compare this information over the same amount of time? Don't forget apps usually are open in the background for longer periods than tabs would be on a browser. How about compare 12 hours (or so) of each and see what difference there is?

**Comment by schr3j** [comment]
Well this was within the first 34s after opening the app... 
1,7MB sent and 76,8MB received.
Maybe there are some problems with the cache or fonts in general? I don't think there are fonts to load that are above a few MBs...
![bildschirmfoto 2015-01-25 um 17 57 55](https://cloud.githubusercontent.com/assets/1710904/5892343/c5338fa4-a4bb-11e4-85fb-f745c0e35562.png)

**Comment by cameronbracken** [comment]
The hostname that is sucking the bandwidth for me is csi.gstatic.com which is constantly pulling down about 1 MB/s

**Comment by chenasraf** [comment]
I'll look into webkit caching, maybe it has something to do with that. I'll also try looking at what the file actually holds, if it's truly static, maybe I can host it locally. Thanks for the input everyone.

**Comment by gboudreau** [comment]
Do you guys still experience this? Downloaded the latest, and I'm not seeing much (less than 8MB for about 10 minutes open and playing with it).
I'm checking in Activity Monitor. (Does Activity Monitor report the same numbers?)

**Comment by cameronbracken** [comment]
Yes, still sucking down bandwidth with the latest version, about 100MB in 2 minutes, Activity monitor reports about 70MB in the same time.

**Comment by davidnknight** [comment]
@cameronbracken @schr3j Apologies for the off topic post but what is that app you're both using?

**Comment by cameronbracken** [comment]
The most recent version here: https://github.com/chenasraf/gInbox/releases

**Comment by davidnknight** [comment]
@cameronbracken So that modal that shows connections etc is part of this app?

**Comment by cameronbracken** [comment]
Oh I misunderstood, it's [LittleSnitch](https://www.obdev.at/products/littlesnitch/index.html).

**Comment by cameronbracken** [comment]
Sidenote, I've been monitoring gInbox traffic for the last hour and its only pulled down 10MB, looks like the issue may be resolved.
